scripts/parse_proceedings.py

```diff
@@ -1,175 +0,0 @@
-import argparse
-import csv
-import json
-import re
-import subprocess
-
-import requests
-from bs4 import BeautifulSoup
-
-
-def parse_arguments():
```

Comment:
These scripts are used for generating the web content (e.g. the indexes of papers, talks etc). Presumably they're just temporary being deleted? Will they be used in future? If so, perhaps easiest to keep them?

Reply:
Yes, this PR is to get a clean start with some minor fixes. Depending on content/data this year we can always add this back easily as it it saved in the archive branch.